addon/components/modal.js

```diff
@@ -148,7 +148,7 @@ export default Component.extend({
 		}
 
 		const scheduler = this.get('scheduler');
-		const element = this.$().get(0);
+		const element = this.element;
```

Comment:
`this.get('element')` ?

Reply:
Non computed property, no need for `get` method.